apps/remix-ide/src/app/tabs/compile-tab.js

```diff
@@ -2,7 +2,7 @@
 import { ViewPlugin } from '@remixproject/engine-web'
 import * as packageJson from '../../../../../package.json'
 import publishToStorage from '../../publishToStorage'
-import { compile } from '../compiler/compiler-helpers'
```

Comment:
Will your changes to import path and plugin calls affect my react solidity compiler move PR?

Reply:
I think so yes. I'm going to move everything now to @remix-project/core-plugin as suggested by @yann300